src/mfront.jl

```diff
@@ -0,0 +1,60 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/Materials.jl/blob/master/LICENSE
+
+mbv = MFrontInterface.behaviour
+
+mutable struct MFront <: AbstractMaterial
```

Comment:
I don't know if I would create a struct per behavior. This seems quite cumbersome, isn't it ?
If you do want to mix the behavior and a the behavior data in the same structure for convenience, that's ok as long as the behavior object is not duplicated at each Gauss point (this would be a huge memory penalty).

Reply:
I think the goal is to make a general approach that we could cover all behaviors. I don't know how to do it exactly. I will need to do something working first that there is something to refactor.